src/app/signers/SignersTable.tsx

```diff
@@ -0,0 +1,266 @@
+import { useColorModeValue } from '@chakra-ui/react';
+import styled from '@emotion/styled';
+import { UseQueryResult, useQueries, useQueryClient } from '@tanstack/react-query';
+import React, { ReactNode, useMemo, useState } from 'react';
+
+import { AddressLink } from '../../common/components/ExplorerLinks';
+import { Section } from '../../common/components/Section';
+import { ApiResponseWithResultsOffset } from '../../common/types/api';
+import { truncateMiddle } from '../../common/utils/utils';
+import { Flex } from '../../ui/Flex';
+import { Show } from '../../ui/Show';
+import { Table } from '../../ui/Table';
+import { Tbody } from '../../ui/Tbody';
+import { Td } from '../../ui/Td';
+import { Text } from '../../ui/Text';
+import { Th } from '../../ui/Th';
+import { Thead } from '../../ui/Thead';
+import { Tr } from '../../ui/Tr';
+import { ScrollableBox } from '../_components/BlockList/ScrollableDiv';
+import { useSuspenseCurrentStackingCycle } from '../_components/Stats/CurrentStackingCycle/useCurrentStackingCycle';
+import { SortByVotingPowerFilter, VotingPowerSortOrder } from './SortByVotingPowerFilter';
+import { mobileBorderCss } from './consts';
+import { SignersStackersData, useGetStackersBySignerQuery } from './data/UseSignerAddresses';
+import { SignerInfo, useSuspensePoxSigners } from './data/useSigners';
+
+const StyledTable = styled(Table)`
```

Comment:
I personally like using styled components, and considering that we installed emotion, I think it's fair game to use them. In this particular case, I went with the styled component because I think it conveys my styling intention which is to remove the border-bottoms from the header and last row succintly, as opposed to spreading out this logic across different components and requiring JS logic in order to be able to apply the styling